docs/test/misc/ds6/alignment-controls-inline.html

```diff
@@ -1,62 +1,66 @@
 ---
 layout: ds6/test
-title: Inline Alignment of Textbox, Button, Fake Button, Select, Combobox, Listbox &amp; Menu
+title: Inline Alignment of Textbox, Button, Fake Button, Select, Combobox, Listbox Button &amp; Menu
 ---
 
-<h2>Small (only available with buttons)</h2>
-<div class="test" style="padding: 1px">
+<h2>Small</h2>
+<div class="test">
     <span class="textbox textbox--small">
         <input class="textbox__control" placeholder="placeholder text" type="text" />
     </span>
     <span class="select select--small">
-        <select>
+        <select class="select__control">
             <option value="1">Option 1</option>
             <option value="2">Option 2</option>
             <option value="3">Option 3</option>
         </select>
         <span class="select__icon"></span>
     </span>
     <button type="button" class="btn btn--primary btn--small btn--small-fixed-height">Button</button>
-    <a class="fake-btn fake-btn--primary fake-btn--small fake-btn--small-fixed-height" href="#">Fake Button</a>
-    <span class="combobox">
+    <a class="fake-btn fake-btn--regular fake-btn--small fake-btn--small-fixed-height" href="#">Fake Button</a>
+    <span class="combobox combobox--small">
```

Comment:
There's no `.combobox--small` in the DS, unless I'm mistaken. Also, we don't have it anywhere in the LESS. We should just leave out this class.

Reply:
Okay, thanks. The idea was to at least have them all side by side so we can see how they look and spot problems with the DS like this. As per description, I plan on reviewing this situation in a follow up PR. We definitely need to sort out this situation so I don't just want to hide things for now.

Reply:
True. It's just that the DS isn't prescriptive in this area, and (even though this is a test page) people might look and think there should be a `--small` version of the combobox.

Reply:
My plan is to have this representing the "truth" before we release. So nobody can see it now. Just us, which helps us fix it up.

Reply:
Once @agliga has the menu changes in, I will revisit this.

Reply:
Well, `combobox--small` isn't truth. That's what I'm saying. We don't have that selector.

Reply:
That's why I was using future tense. I plan to have it representing the truth before we release.

Reply:
But I still don’t know why it was added if the corresponding class wasn’t added?

Reply:
Good point. Rather than go through and update the markup again in the ds6 page, I merely copied over all of the HTML from the DS4 test page.

Reply:
Can I move forward with this PR? The test page will be addressed again when all the pieces are in place (your changes, andrew's and mine).

Reply:
I know it's minor, but it might be forgotten. ;-)